cfme/tests/configure/test_access_control.py

```diff
@@ -1464,6 +1506,83 @@ def test_superadmin_tenant_admin_crud(appliance):
     assert not user.exists
 
 
+@pytest.mark.tier(2)
+@test_requirements.multi_tenancy
+def test_tenantadmin_user_crud(new_tenant_admin, request, appliance):
+    """
+    As a Tenant Admin I want to be able to create users in my tenant
+
+    Polarion:
+        assignee: nachandr
+        casecomponent: Configuration
+        caseimportance: high
+        tags: cfme_tenancy
+        initialEstimate: 1/4h
+        startsin: 5.5
+        testSteps:
+            1. Login as super admin and create new tenant
+            2. Create new role by copying EvmRole-tenant_administrator
+            3. Create new group and choose role created in previous step and your
+            tenant
+            4. Create new tenant admin user and assign the user to the group created in
+            the previous step
```

Comment:
Steps 1-4 look like they are part of setup, as they are taken care of in a fixture.